packages/wallets/src/features/cashier/modules/TransactionStatus/hooks/useRecentTransactions.ts

```diff
@@ -0,0 +1,168 @@
+import { useEffect, useMemo } from 'react';
+import moment from 'moment';
+import { useCryptoTransactions } from '@deriv/api';
+
+const useRecentTransactions = () => {
```

Comment:
There is too much happening in this hook. Can we break it down ?

Reply:
Have moved mappers to a separate file